src/pudl/transform/params/ferc1.py

```diff
@@ -3554,6 +3554,224 @@
             },
         ],
     },
+    "electric_oandm_ferc1": {
+        "rename_columns_ferc1": {
+            "dbf": {
+                "columns": {
+                    "respondent_id": "utility_id_ferc1_dbf",
+                    "report_year": "report_year",
+                    "report_prd": "report_prd",
+                    "row_prvlg": "row_prvlg",
+                    "row_number": "row_number",
+                    "row_seq": "row_seq",
+                    "spplmnt_num": "spplmnt_num",
+                    "crnt_yr_amt": "annual_oandm_expense",
+                    "xbrl_factoid": "oandm_expense_type",
+                }
+            },
+            "xbrl": {
+                "columns": {
+                    "entity_id": "utility_id_ferc1_xbrl",
+                    "report_year": "report_year",
+                    "xbrl_factoid": "oandm_expense_type",
+                }
+            },
+            "duration_xbrl": {
+                "columns": {
+                    xbrl_col: f"{xbrl_col}_annual_oandm_expense"
```

Comment:
lol this list of cols is so long! i'm looking forward to adding a `add_suffix` transformer that enables us to not spell out all of the column names in this rename-fashion when we just want to add one suffix to all but a few columns. Not implemented yet and I'm not suggesting you do it rn. but I'm looking forward to it!

Reply:
yaaa me too!